fix9.sh

```
@@ -0,0 +1 @@
+sed -i 's/if not re.match(r'\''\^\[a-zA-Z0-9.-\]+.*/if not re.match(r"^[a-zA-Z0-9.-]+$", str(host)):/g' setup/services.py
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)